crates/sui-types/src/error.rs

```diff
@@ -248,6 +248,12 @@ pub enum UserInputError {
 
     #[error("Immutable parameter provided, mutable parameter expected.")]
     MutableParameterExpected { object_id: ObjectID },
+
+    #[error("Address {address} is denied for coin defined in {coin_package_id:?}")]
+    AddressDeniedForCoin {
```

Comment:
where is this used?

Reply:
It will be used in the next PR.

Reply:
Ideally could be moved to the PR where it will be used so it's not dangling, but not a huge deal if it's inconvenient